bear.java

```diff
@@ -0,0 +1,57 @@
+//public class bears extends animals {
+  //  public String _name;
+
+class bears extends animals {
+    //super(name); 
+    //}
+
+  //public String species() {
+        //return "bears";
+    //}
+  int Intel = 6;
```

Comment:
This property is already defined in the base class `Animal`.  You just need a way to set the value rather than defining your in this class and duplicated all the code from Animal.  The point of inheritance is so put all the common code in one place and not repeat code everywhere!